### ISSUE TITLE
NullReferenceException in GetStencilDepth

### ISSUE THREAD

**Issue by andreierdoss** [Flafla2/Unity-Wiimote#5]
Hello,

I am very excited to see this library. Thank you for your hard work. I opened up the repo in Unity 5.2.1f1 on Mac Yosemite and the Scene when I noticed this error showed up.

```
NullReferenceException: Object reference not set to an instance of an object
UnityEngine.UI.MaskUtilities.GetStencilDepth (UnityEngine.Transform transform, UnityEngine.Transform stopAfter) (at /Users/builduser/buildslave/unity/build/Extensions/guisystem/UnityEngine.UI/UI/Core/MaskUtilities.cs:73)
UnityEngine.UI.MaskableGraphic.GetModifiedMaterial (UnityEngine.Material baseMaterial) (at /Users/builduser/buildslave/unity/build/Extensions/guisystem/UnityEngine.UI/UI/Core/MaskableGraphic.cs:71)
UnityEngine.UI.Graphic.get_materialForRendering () (at /Users/builduser/buildslave/unity/build/Extensions/guisystem/UnityEngine.UI/UI/Core/Graphic.cs:227)
UnityEngine.UI.Graphic.UpdateMaterial () (at /Users/builduser/buildslave/unity/build/Extensions/guisystem/UnityEngine.UI/UI/Core/Graphic.cs:327)
UnityEngine.UI.Graphic.Rebuild (CanvasUpdate update) (at /Users/builduser/buildslave/unity/build/Extensions/guisystem/UnityEngine.UI/UI/Core/Graphic.cs:311)
UnityEngine.UI.CanvasUpdateRegistry.PerformUpdate () (at /Users/builduser/buildslave/unity/build/Extensions/guisystem/UnityEngine.UI/UI/Core/CanvasUpdateRegistry.cs:107)
```

I managed to run the app and connect to the Wiimote. It showed that the buttons were pressed but the Wiimote was not moving. Afterwards I stopped running the game, and the screen went white and Unity stopped responding.

After some more digging it seems like this is related to the upgrade to 5.2. Please see this link: http://answers.unity3d.com/questions/1064569/upgrading-to-unity-52-all-image-script-components.html  
Adding an Image Component got rid of these errors. I didn't do a pull request because the Image component has a white color, thus I wasn't sure what it is supposed to be.

**Comment by Flafla2** [comment]
Hi,

Thanks for your work and investigation into this.  I hate Unity sometimes, all I want is a simple mask without a background image!  I'll look into this in a few days, right now I am swamped with college applications.  I don't think it should be too hard of a fix though.

**Comment by Grogath** [comment]
Hi, I solved this problem by adding an empty Image Component on the "BB" object (the parent) ! It seems that since 5.2 Unity asks that for some obscure reason !

**Comment by Flafla2** [comment]
I am going to close this, as the issue was resolved in Unity version 5.2.2:

```
(none) - UI: Image will now use the white texture again if no texture is specified.
```

https://unity3d.com/unity/whats-new/unity-5.2.2
